system/helpers/dateUtils.cfm

```diff
@@ -3,4 +3,90 @@
 	<cfargument name="timezone" type="string" required="false" default="GMT" /><cfsilent>
 
 	<cfreturn DateFormat( arguments.date, "ddd, dd MMM YYYY " ) & TimeFormat( arguments.date, "HH:mm:ss " ) & arguments.timezone />
+</cfsilent></cffunction>
+
+<cffunction name="getShortDateFormatMask" access="public" returntype="string" output="false"><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getShortDateFormatMask() />
+</cfsilent></cffunction>
+
+<cffunction name="getShortDate" access="public" returntype="string" output="false">
+	<cfargument name="date" type="date" required="true" /><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getShortDate( arguments.date ) />
+</cfsilent></cffunction>
+
+<cffunction name="getLongDateFormatMask" access="public" returntype="string" output="false"><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getLongDateFormatMask() />
+</cfsilent></cffunction>
+
+<cffunction name="getLongDateFormatDayMask" access="public" returntype="string" output="false"><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getLongDateFormatDayMask() />
+</cfsilent></cffunction>
+
+<cffunction name="getLongDateFormatMonthMask" access="public" returntype="string" output="false"><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getLongDateFormatMonthMask() />
+</cfsilent></cffunction>
+
+<cffunction name="getLongDateFormatYearMask" access="public" returntype="string" output="false"><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getLongDateFormatYearMask() />
+</cfsilent></cffunction>
+
+<cffunction name="getLongDateFormatDayMonthMask" access="public" returntype="string" output="false"><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getLongDateFormatDayMonthMask() />
+</cfsilent></cffunction>
+
+<cffunction name="getLongDate" access="public" returntype="string" output="false">
+	<cfargument name="date" type="date" required="true" />
+	<cfargument name="includeOrdinal" type="boolean" required="false" default="false" /><cfsilent>
+	<cfset var dfService = getSingleton( "dateFormatService" ) />
+	<cfreturn dfService.getLongDate( arguments.date, arguments.includeOrdinal ) />
```

Comment:
In all these, I'd be inclined to pass
`( argumentCollection=arguments )`
It's simpler, and it will also pass them all as named arguments to the service...